src/operators/skipWhile.ts

```diff
@@ -0,0 +1,47 @@
+import {Operator} from '../Operator';
+import {Subscriber} from '../Subscriber';
+import {tryCatch} from '../util/tryCatch';
+import {errorObject} from '../util/errorObject';
+import {bindCallback} from '../util/bindCallback';
+
+export function skipWhile<T>(predicate: (x: T, index: number) => boolean, thisArg?: any) {
+  return this.lift(new SkipWhileOperator(predicate, thisArg));
+}
+
+class SkipWhileOperator<T, R> implements Operator<T, R> {
+  predicate: (x: T, index: number) => boolean;
+
+  constructor(predicate: (x: T, index: number) => boolean, thisArg?: any) {
+    this.predicate = <(x: T, index: number) => boolean>bindCallback(predicate, thisArg, 2);
+  }
+
+  call(subscriber: Subscriber<T>): Subscriber<T> {
+    return new SkipWhileSubscriber(subscriber, this.predicate);
+  }
+}
+
+class SkipWhileSubscriber<T> extends Subscriber<T> {
+  private skipping: boolean = true;
+  private index: number = 0;
+
+  constructor(destination: Subscriber<T>,
+              private predicate: (x: T, index: number) => boolean) {
+    super(destination);
+  }
+
+  _next(value: T): void {
+    const destination = this.destination;
```

Comment:
Like for everything else, you could store `const skipping = this.skipping` since it's used multiple times.